### ISSUE TITLE
[Suggestion] Alexa app?

### ISSUE THREAD

**Issue by WiiBii** [Monika-After-Story/MonikaModDev#7353]
I know this might be a long shot, but maybe later in development you could do some sort of Alexa app that ties to your Monika? If not direct interaction, maybe she would just take note of the questions you ask or the music you listen to and bring it up in conversation?

**Comment by Mugnika** [comment]
that's a very VERY long shot.

**Comment by WiiBii** [comment]
Yeah that's why I was thinking of it more as an endgame project, especially because it would be using more adaptive code. Maybe instead Monika would be able to read your Spotify music, possibly through your Discord status? I'm just throwing things out there, of course!

**Comment by xXMidnight** [comment]
> Yeah that's why I was thinking of it more as an endgame project, especially because it would be using more adaptive code. Maybe instead Monika would be able to read your Spotify music, possibly through your Discord status? I'm just throwing things out there, of course!

It's possible, but I - kind of? - know how to do it only in HTML. The Python I'm used to is the turtle logo kind, eg:

`tom.fd(100)`
`for count in range (6):`
`    tom.lt(360/6)`
`    tom.fd(100)`

that kind of Python. So uh, sorry, but I can't help here -w-"













that is a **seriously long, _LONG_** shot my guy

**Comment by ThePotatoGuy** [comment]
Confused on what is being suggested here.

>maybe she would just take note of the questions you ask or the music you listen to and bring it up in conversation

Is like Monika bringing it up on an amazon echo? Or MAS itself acting like an alexa?

>Maybe instead Monika would be able to read your Spotify music, possibly through your Discord status

This wouldnt be too bad assuming:
1. APIs exist for getting this info
2. permissions to access said info can be easily retrieved or requested from the user.
